tests/test_schemas.py

```diff
@@ -616,8 +616,82 @@ def test_get_node_range(self):
     def test_get_node_required(self):
         return
 
-    def test_get_node_validation_rules(self):
-        return
+    @pytest.mark.parametrize(
```

Comment:
If it isn't too much trouble to extract this out to the `tests/integration` let's move forward on adding new items in that directory.

Reply:
hmm I think this is a unit test since I am only testing `get_node_validation_rules` function, and integration tests typically involve more than one function?

Reply:
Unit tests need to be able to run on an air-gapped machine. Meaning that it does not depend on any outside connection to the world. Within the python client we are enforcing this by disabling networking: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/tests/unit/conftest.py#L18-L35


For this - If it does connect to any other services then it is an integration test.

If it does not connect to any other services then it's a bit subjective and does fall into your idea of "involves more than one function", but not always.

Reply:
@BryanFauble Thanks for pointing this out. I tried adding the following: 
```
        from pytest_socket import SocketBlockedError, disable_socket
        disable_socket(allow_unix_socket=True)
```
into the middle of my test `test_get_node_validation_rules_valid`, and it still passed. I think this means that this test is indeed a unit test :)

Reply:
Let me add this into the `unit` folder then.